src/pages/MonthlyGood/LandingForm.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+	<form @submit.prevent.stop="submit" novalidate>
+		<fieldset>
+			<label class="input-label" :class="{ 'error': $v.$invalid }" for="amount">
+				<kv-currency-input id="amount" :value="amount" @input="updateAmount" />
```

Comment:
I think we'll want to have `<kv-currency-input>` outside of the the `<label>` element since we're the for and id attributes.
We'll also want to have some text inside the label even if it's visually hidden for screen readers to pick up.